mne/channels/channels.py

```diff
@@ -439,13 +439,13 @@ class UpdateChannelsMixin(object):
     def pick_types(self, meg=True, eeg=False, stim=False, eog=False,
                    ecg=False, emg=False, ref_meg='auto', misc=False,
                    resp=False, chpi=False, exci=False, ias=False, syst=False,
-                   seeg=False, bio=False, ecog=False, include=[],
-                   exclude='bads', selection=None):
+                   seeg=False, dipole=False, gof=False, bio=False, ecog=False,
+                   include=[], exclude='bads', selection=None):
```

Comment:
Oh wait that's not how probability works... I mean something more like `1-(1-1e-12) ** 8` is still small. Now if the original probability is more like `1e-1` we're in more trouble, but I don't think it's so high.